docs/pages/CLITemplate.md

```diff
@@ -0,0 +1,118 @@
+# UI5 CLI
+## Use the UI5 command line
```

Comment:
I would remove this headline (it doesn't add much information) and lift all following headlines one level up. Currently, the command names are on level four (`####`) and can't be differentiated from bold text.  
I think level three would be more suitable for those.